examples/demo_mapviewer_features.ipynb

```diff
@@ -114,6 +114,45 @@
     "# center the map on a specific substation\n",
     "network_map2.center_on_substation('P1')"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Example 3\n",
+    "# creates a network that contains lines and hvdc lines, and adds a substation and a tie line.\n",
+    "# Then, adds some geographical data extensions, not present in the original network (fictitious coordinates).\n",
+    "# Finally, display the network using the mapviewer widget\n",
+    "\n",
+    "four_substations_network = pn.create_four_substations_node_breaker_network()\n",
+    "\n",
+    "import pandas as pd\n",
+    "four_substations_network.create_substations(id='S5')\n",
+    "four_substations_network.create_voltage_levels(id='S5VL1', substation_id='S5', topology_kind='BUS_BREAKER', nominal_v=400)\n",
+    "four_substations_network.create_buses(id='S5VL1B1', voltage_level_id='S5VL1')\n",
+    "four_substations_network.create_dangling_lines(id='d1', voltage_level_id='S4VL1', node=1, p0=10, q0=3, r=0, x=5, g=0, b=1e-6)\n",
+    "four_substations_network.create_dangling_lines(id='d2', voltage_level_id='S5VL1', bus_id='S5VL1B1', p0=10, q0=3, r=0, x=5, g=0, b=1e-6)\n",
+    "four_substations_network.create_tie_lines(id='t1', dangling_line1_id='d1', dangling_line2_id='d2')\n",
+    "voltage_levels = four_substations_network.get_voltage_levels()\n",
+    "voltage_levels['name']=voltage_levels.index\n",
+    "coords = pd.DataFrame(\n",
+    "    [\n",
+    "        [voltage_levels.loc[voltage_levels[\"name\"] == \"S1VL1\", \"substation_id\"].iloc[0], 2.9653506942899255, 46.648820589226624],\n",
```

Comment:
Is it possible to switch the longitude and latitude columns when creating the `coords` Dataframe? If a user uses the example with their own coordinates, they may put latitude before longitude because it seems more usual.